docs/dev/table/hive/index.md

```diff
@@ -303,5 +303,6 @@ Currently `HiveCatalog` supports most Flink data types with the following mappin
 Note that:
 
 * Flink doesn't support Hive's `UNION` type is not supported
+* Hive's `TIMESTAMP` always has precision 9 and doesn't support other precisions. As a result, `HiveCatalog` cannot store `TIMESTAMP` columns whose precisions are not 9. Hive UDFs, on the other hand, can process `TIMESTAMP` values with a precision <= 9.
```

Comment:
> As a result, `HiveCatalog` cannot store `TIMESTAMP` columns whose precisions are not 9

sorry if I misunderstand it as the conversation has been quite complex so far - should be ok to store precision less or equal to 9, right? if it's say 6, users don't loose any data, we can just append more units to the original data, no?

Reply:
No we can't store timestamp(<9) to HMS because the precision will be lost, and when we retrieve the column from HMS it'll always be timestamp(9) which can lead to problems.